robottelo/common/ssh.py

```diff
@@ -167,7 +170,7 @@ def command(cmd, hostname=None, output_format=None, timeout=None):
         stdout = stdout.replace('""', '')
         stdout = u''.join(stdout).split('\n')
         stdout = [
-            regex.sub('', line) for line in stdout if not line.startswith('[')
+            regex.sub('', line) for line in stdout if not line.startswith("[")
```

Comment:
This should be reverted back. We prefer single quotes over double quotes for strings. But double quotes can be used on some cases, and this is not the case.